### ISSUE TITLE
Documentation suggestion

### ISSUE THREAD

**Issue by morphiak** [mcdope/pam_usb#138]
I just had quite a headache trying to get started because I was unaware of the importance of order in /etc/pam.d/system-auth (Fedora 36) and foolishly put 
`auth    sufficient      pam_usb.so
`
at the bottom of the block as is my personal convention for alterations, rather than above 
`auth    required        pam_unix.so nullok_secure
`
Perhaps altering
`Alter your /etc/pam.d/common-auth configuration to:`
to 
`Alter your /etc/pam.d/common-auth configuration (line order is important) to:`
would stop fools like me from making that mistake in the future. Oh well, learned something anyway, it makes sense now.

Thanks to all for the great work.

**Comment by mcdope** [comment]
Thanks for your feedback!

Yes, pam files are a rule based system. Where each rule can either be sufficient, optional etc. This is nothing pam_usb specific but just basic `pam.d` stuff. 

Imho this is kind of a philosophical thingy. Since a manual installation requires users to mess with quite important auth configuration files it's assumed "if they dare too, they will read the documentation on how pam works" - esp. since common-auth is only the right place for Debian based distros afaik (for Fedora there are other files given in https://github.com/mcdope/pam_usb/wiki/Getting-Started#setting-up-the-pam-module). This is also since messing with the files can easily make you unable to login at all, requiring recovery mode. 

But then again, the docs doesn't point out one should read into pam.d first. Also it wouldn't be much effort... I will add both, a paragraph about reading into how these files works first and your suggested braces.

Thanks for the suggestion.

**Comment by mcdope** [comment]
Added. See https://github.com/mcdope/pam_usb/wiki/Getting-Started/_compare/0f4997fcf222fedc5b7ca9664685f46a467a0337

**Comment by morphiak** [comment]
Yeah, totally get the if you're not up to working it out it's probably not for you philosophy, especially in the guts of security. I self debated bringing it up for those reasons, also because after kicking around linux for a couple of decades I have inevitably learned that before and forgotten it and am a bit ashamed.

I do think with a new influx of linux users from the steam deck, and with a lot of people with machines not compatible with Win11 probably incoming soon, it's time to be a bit more explicit in these things. Glad to see you're that way inclined too.

Have a good one.
